.travis.yml

```diff
@@ -11,27 +11,33 @@ os:
   - linux
   - osx
 
+sudo: required # needed for trusty
+dist: trusty # needed for Raqm
+
 before_install:
   - if [ "$TRAVIS_OS_NAME" = "linux" ];
     then
+      sudo apt-add-repository -y ppa:as-bahanta/raqm;
```

Comment:
May be it is better to use a matrix and build once with and once without raqm, that would make sure the Raqm path does not bitrot? (note that the Mac build doesn’t use Raqm, so currently both code paths are exercised, though on different platforms).